juju/juju
# Add instance.id to CharmProfilingInfo

## Description of change

The following changes adds the CharmProfilingInfo to the api and
apiserver. Additionally it also adds instance.Id to the machine
to ensure that we follow the same semantics (we're congruent) with
the state machine.

## QA steps

This is a mechanical change, and should be noted that running the unit
tests should ensure that the instance.Id is passed through.

Additionally you can alter the `worker/instancemutator/mutater.go` file to
log out the `InstanceId` from the `CharmProfilingInfo`.
